internal/web_server/web/views/pages/dashboard_pages/user_admin_page.templ

```diff
@@ -0,0 +1,19 @@
+package dashboard_pages
+
+import (
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/layout_components"
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/dashboard_components/user_admin"
+	"github.com/Molnes/Nyhetsjeger/internal/models/users/usernames"
+)
+
+
+// The username administration page main function
+templ UsernameAdminPage(data *usernames.UsernameAdminInfo) {
+	@layout_components.DashBoardLayout("Toppliste") {
```

Comment:
Name of page should probably not be toppliste

Reply:
changed from placeholder name to more final name